### PR TITLE
Fix/catkin make

### PR DESCRIPTION
**Pull request by supertoto29**
**Warning :** Before creating a pull request, make sure that it does not already exists in the [pull requests](../). Thank you.

## Description
Fix the develop container and completed the read thread.

## How has this been tested ?
Tested locally with the sensor

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: .github/workflows/docker-image-perception-fix.yml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    if: failure()
+    steps:
+      - name: Notify Slack Fail
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+        uses: voxmedia/github-action-slack-notify-build@v1
+        with:
+          channel: github-ci-notifications
+          status: FAILED
+          color: danger
```
Comment (Kam0ss) [inline .github/workflows/docker-image-perception-fix.yml:113]: This is useless.

**Review by Kam0ss — CHANGES_REQUESTED** [review]
LGTM, one change to do.

File: .github/workflows/docker-image-perception-fix.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
+name: Docker Image CI - Develop Branch
```
Comment (Kam0ss) [inline .github/workflows/docker-image-perception-fix.yml:1]: Should be Fix instead of Develop.

Comment (supertoto29) [reply .github/workflows/docker-image-perception-fix.yml]: Fix made

**Review by 0x72D0 — APPROVED** [review]
LGTM!

**Pull request merged**